internal/interfaces/grpc/service_one_port.go

```diff
@@ -339,7 +340,24 @@ func (s *serviceOnePort) newServer(
 		grpcweb.WithOriginFunc(func(origin string) bool { return true }),
 	)
 
-	handler := router(grpcServer, grpcWebServer, grpcGateway)
+	tdexConnectSvc, err := httpinterface.NewTdexConnectService(
+		s.opts.AppConfig.WalletService().Wallet(),
+		s.validatePassword,
+		adminMacaroonPath,
```

Comment:
why do we need macaroon passed in the tdexconnect service?

Reply:
i didn't change this, it's inherited from master https://github.com/tdex-network/tdex-daemon/blob/master/internal/interfaces/grpc/service.go#L357-L364